### PR TITLE
Python bindings for native Windows

### PR DESCRIPTION
**Pull request by ampli**
This is the initial release of Python bindings for Windows.
It includes bindings for Python2 and Python3.
It also includes a Python script to set the environment for running Python scripts with LG bindings.
This script runs the tests by default.
No test fails.
3 tests are automatically skipped (2 for spell and one of SAT - no support yet).

(Some of the commits here are duplicate from a pending PR.)

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -35,7 +35,9 @@ CONTENTS of this directory:
    bindings/lisp/*          Optional Common Lisp language bindings.
    bindings/ocaml/*         Optional OCaML language bindings.
    bindings/python/*        Optional Python2 language bindings.
-   bindings/python3/*       Optional Python3 language bindings.
+   bindings/python3/*       Optional Python3 languagSe bindings.
```
Comment (linas) [inline README]: languagSe  ?

File: bindings/python-examples/README.md [hunk 2]
```diff
@@ -1,38 +1,103 @@
...
+-----------
+A Link Grammar library test is implemented in **tests.py**.
+An example program **example.py** is provided.
 
+Configuring (if needed)
+-----------------------
+### For Python2
+   $ configure --enable-python-bindings
+### For Python3
+   $ configure --enable-python4-bindings
```
Comment (linas) [inline bindings/python-examples/README.md]: python4 ??

File: bindings/python-examples/README.md [hunk 3]
```diff
@@ -1,38 +1,103 @@
...
+      +---Wd---+-Ss*b+  |     +----A----+     |
+      |        |     |  |     |         |     |
+  LEFT-WALL this.p is.v a simple.a sentence.n .
+```
 Additional examples can be found in `examples.py`.
+
+Testing
+-------
+The test collection **tests.py** should run 56 tests, none of them should fail.
+However, 3 tests will get skipped if the library is not configured with a speller, and one test will get skipped if the library is not configured with the SAT solver (this is the status for now on native Windows).
```
Comment (linas) [inline bindings/python-examples/README.md]: please wrap to 80-columns

**Comment by linas** [comment]
If you push one more commit to this same branch, with the trivial typo fixes in it, then you won't need to create a new pull request. I'll wait to merge, just to keep it simple.

**Comment by ampli** [comment]
I pushed again the fixed branch (-f). I folded the lines, and I can see the fixed README.md, but when I click on **View** I see the unfolded one. I guess this is a non-harmful GitHub bug.

(I originally didn't fold the lines because I found that with folding this README.md looks bad, because lines with boldface words can be much shorter.)

**Comment by ampli** [comment]
How come it got conflicts?

**Comment by linas** [comment]
Because I merged #363 first, and this one followed that one. I am kind-of surprised, since that usually works.   I get the impression that maybe github somehow changed the way that they handle pull requests, because these things used to work smoothly a year ago, and then started going haywire right around the time that the "continuous integration" stuff got turned on.

**Pull request merged**